akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/StashSpec.scala

```diff
@@ -28,8 +28,8 @@ object AbstractStashSpec {
   final case class GetStashSize(replyTo: ActorRef[Int]) extends Command
 
   val immutableStash: Behavior[Command] =
-    Behaviors.setup[Command] { _ =>
-      val buffer = StashBuffer[Command](capacity = 10)
+    Behaviors.setup[Command] { ctx =>
+      val buffer = StashBuffer[Command](ctx, capacity = 10)
```

Comment:
change to `Behaviors.withStash`?

Reply:
:+1: missed all the uses in this test. Might be nice in the future to have pure user facing API tests not in the same package so this wouldn't have compiled